cron/poll_read_issue_comments.py

```diff
@@ -135,7 +157,63 @@ def get_command_votes(api, urn, comment_id):
     return votes
 
 
-def handle_vote_command(api, command, issue_id, comment_id, votes):
+def get_meritocracy():
+    # FIXME: update when this is done by the db
```

Comment:
I'm not sure the meritocracy will ever be stored in the DB. Instead, put [this code](https://github.com/chaosbot/Chaos/blob/e0e352896eddec03c01ba4e85b1263b83bb669f0/cron/poll_pull_requests.py#L40-L51) in the function.